applications/FluidDynamicsApplication/python_scripts/apply_inlet_process.py

```diff
@@ -8,22 +8,24 @@ def Factory(settings, Model):
         raise Exception("expected input shall be a Parameters object, encapsulating a json string")
     return ApplyInletProcess(Model, settings["Parameters"])
 
+def GetHistoricalVariables(settings):
+    copy_settings = settings.Clone()
+    copy_settings.AddMissingParameters(ApplyInletProcess.GetDefaultParameters())
+    model_part_name = copy_settings["model_part_name"].GetString()
+    if "." in model_part_name:
+        root_model_part = model_part_name[:model_part_name.index(".")]
+    else:
+        root_model_part = model_part_name
```

Comment:
```suggestion
    model_part_name = copy_settings["model_part_name"].GetString().split(".")[0]
```

this should work in any case

I will do a more detailed review later